js/client.js

```diff
@@ -1648,7 +1665,12 @@
 				autojoinCount++;
 				if (autojoinCount >= 10) break;
 			}
-			Tools.prefs('autojoin', autojoins.join(','));
+			if (typeof curAutojoin !== 'string') {
+				curAutojoin[Config.server.id] = autojoins.join(',');
```

Comment:
`Tools.prefs('autojoin');` exits without saving the data if there is no change in the roomlist.